package.json

```diff
@@ -79,9 +83,16 @@
       "<rootDir>/tests/**/*.spec.ts"
     ]
   },
+  "lint-staged": {
+    "{,src/**/}*.{md,json,ts}": [
```

Comment:
add also html extension.
Remove the `{}` around `{,src/**/`

Reply:
`{,src/**/}*` is to match files in the current directory (see comma before `src`) and in `src` directory.